index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script src="index.js" defer></script>
+</head>
+<body>
+<section>
+    <img src="map.png">
+    <div class="container">
+        <button id="startGame">Start Game!</button>
+        <button id="instructions">Instructions</button>
```

Comment:
We need to match the ID for the instructions button

Reply:
myBtn changed to instructions - ready to merge